### PR TITLE
jqueryui.com issue #94

### PR DESCRIPTION
**Pull request by ashg1910**

### CODE REVIEW

File: themes/jquery/js/main.js [hunk 1]
```diff
@@ -356,6 +356,12 @@ $(function() {
 			plugin = parts[ 3 ],
 			demo = parts[ 4 ].substring( 0, parts[ 4 ].length - 5 );
 
+		var str1 = "default";
+		var str2 = "effect";
+		if ( demo.localeCompare(str1) == 0 && plugin.localeCompare(str2) == 0 ) {
```
Comment (scottgonzalez) [inline themes/jquery/js/main.js]: This is a fairly roundabout way to do this check, why not just `plugin === "effect" && demo === "default"`?

Comment (ashg1910) [inline themes/jquery/js/main.js]: I thought this would be easily understandable to others

Comment (scottgonzalez) [inline themes/jquery/js/main.js]: This is much less understandable than the direct comparison.

Comment (ashg1910) [inline themes/jquery/js/main.js]: Should I change it to plugin === "effect" && demo === "default"?

Comment (arschmitz) [inline themes/jquery/js/main.js]: Deffinitly direct comparison lets not obfuscate this by using locale.

**Comment by ashg1910** [comment]
@scottgonzalez Did the final thing

File: themes/jquery/js/main.js [hunk 2]
```diff
@@ -341,7 +341,7 @@ $(function() {
 /*
  * jqueryui.com
  */
-$(function() {
+$(function() {
```
Comment (arschmitz) [inline themes/jquery/js/main.js]: Extra whites space at end of line. No need to change this line.

Comment (ashg1910) [inline themes/jquery/js/main.js]: @arschmitz sorry that was by mistake :P

**Comment by scottgonzalez** [comment]
Thanks, but while reviewing this I found a better solution that doesn't require hard-coding exceptions into the script.

**Comment by ashg1910** [comment]
@scottgonzalez What is that?

**Comment by scottgonzalez** [comment]
You can see it in the commit that closed this PR: https://github.com/jquery/jquery-wp-content/commit/0196a651368da78cc6186de6d52722b4421f9b4c

**Pull request closed without merging**